integration-tests/deployment/clo/models/models.go

```diff
@@ -0,0 +1,28 @@
+// TODO: KS-455: Refactor this package to use chainlink-common
+// Fork from: https://github.com/smartcontractkit/feeds-manager/tree/develop/api/models
+// until it can be refactored in cahinlink-common.
+
+// Package models provides generated go types that reflect the GraphQL types
+// defined in the API schemas.
+//
+// To maintain compatibility with the default JSON interfaces, any necessary model
+// overrides can be defined this package.
+package models
```

Comment:
Where is the source that models_gen.go was generated from? Is models.go just an addition with some handy constants?